src/components/v5/common/ActionSidebar/partials/forms/ArbitraryTxsForm/hooks.ts

```diff
@@ -0,0 +1,40 @@
+import { Id } from '@colony/colony-js';
+import { useCallback, useMemo } from 'react';
+
+import { ActionTypes } from '~redux/index.ts';
+import { mapPayload } from '~utils/actions.ts';
+import { sanitizeHTML } from '~utils/strings.ts';
+import useActionFormBaseHook from '~v5/common/ActionSidebar/hooks/useActionFormBaseHook.ts';
+import { type ActionFormBaseProps } from '~v5/common/ActionSidebar/types.ts';
+
+import { validationSchema } from './consts.ts';
+
+export const useCreateArbitraryTxs = (
+  getFormOptions: ActionFormBaseProps['getFormOptions'],
+) => {
+  useActionFormBaseHook({
+    actionType: ActionTypes.CREATE_ARBITRARY_TRANSACTION,
+    validationSchema,
+    getFormOptions,
+    defaultValues: useMemo(
+      () => ({
+        createdIn: Id.RootDomain,
+      }),
+      [],
+    ),
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+    transform: useCallback(
```

Comment:
can we try and remove `useCallback` here since it's a weird pattern we have anyways :shrug: 
I've tried removing mapPayload too, but typescript gets really really angry at me :D